source/server-discovery-and-monitoring/tests/README.rst

```diff
@@ -56,7 +56,7 @@ so far. It has the following keys:
 A "server" object represents a correct ServerDescription within the client's
 current TopologyDescription. It has the following keys:
 
-- type: A ServerType name, like "RSSecondary".
+- type: A ServerType name, like "RSSecondary". For multi-threaded implementations, the "PossiblePrimary" type should be interpreted as "RSUnknown".
```

Comment:
I'll also point out that this section is mainly a description of the test format, and isn't the ideal place to include implementation details about SDAM itself.

Reply:
I think this is a useful clarification of the test format itself.

Reply:
Yeah, when implementing it I was unclear whether I should ignore the cases with PossiblePrimary.

Reply:
I'd suggest adding the following to the end of the sentence:

```
(see: [ServerType](../server-discovery-and-monitoring.rst#servertype))
```

Cross-referencing it will at least point back to the canonical source of info for this. Also, it may be helpful to refer to "async and multi-threaded implementations".

Reply:
@lamontnelson do you have an opinion on @jmikola s comment?

Reply:
Sorry, this seems to have slipped through the cracks. I've added "async and multi-threaded implementations" and linked to the specification.